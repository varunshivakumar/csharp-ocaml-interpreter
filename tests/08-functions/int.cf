int a;
int b;


int f() {
  print_str("f\n");
  a++;
  return 10;
}

int pa() {
  print_int(a);
  print_str("\n");
  return -1;
}

a = 1;

pa();
b = f();
print_int(b);
print_str("\n");
pa();
