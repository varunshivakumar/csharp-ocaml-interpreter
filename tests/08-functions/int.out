1
f
10
2
